projects/sonar/src/app/app-routing.module.ts

```diff
@@ -179,7 +179,22 @@ export class AppRoutingModule {
           'organisation',
         ],
         editorLongMode: true,
-        filesEnabled: true,
+        files: {
+          enabled: true,
+          filterList: (item: any) => {
+            return item.metadata && item.metadata.type && item.metadata.type === 'file';
+          },
+          orderList: (a: any, b: any) => {
+            if (a.metadata.order < b.metadata.order) {
+              return -1;
+            }
+            if (a.metadata.order > b.metadata.order) {
+              return 1;
+            }
+            return 0;
```

Comment:
Why not just `a.metadata.order - b.metadata.order` ?

Reply:
Absolutely correct! Thanks